mappings.py

```diff
@@ -0,0 +1,174 @@
+        },
+    }
+}
+
+_MEMBER_MAPPING = {
+    "settings": COMMON_SETTINGS,
+    "mappings": {
+        "properties": {
+            "email": {"type": "keyword"},
+            "hashedEmail": {"type": "keyword", "index": False},
+            "institutionalEmail": {"type": "keyword"},
+            "acceptedTerms": {"type": "boolean"},
+            "isPublic": {"type": "boolean"},
+            "isActive": {"type": "boolean"},
+            "roles": {"type": "keyword"},
+            "title": {"type": "keyword", "index": False},
+            "firstName": {
+                "type": "text",
+                "analyzer": "autocomplete",
+                "fields": {"raw": {"type": "keyword"}},
+            },
+            "lastName": {
+                "type": "text",
+                "analyzer": "autocomplete",
+                "fields": {"raw": {"type": "keyword"}},
+            },
+            "jobTitle": {
+                "type": "text",
+                "analyzer": "autocomplete",
+                "fields": {"raw": {"type": "keyword"}},
+            },
+            "institution": {
+                "type": "text",
+                "analyzer": "autocomplete",
+                "fields": {"raw": {"type": "keyword"}},
+            },
+            "city": {
+                "type": "text",
+                "analyzer": "autocomplete",
+                "fields": {"raw": {"type": "keyword"}},
+            },
+            "state": {
+                "type": "text",
+                "analyzer": "autocomplete",
+                "fields": {"raw": {"type": "keyword"}},
+            },
+            "country": {
+                "type": "text",
+                "analyzer": "autocomplete",
+                "fields": {"raw": {"type": "keyword"}},
+            },
+            "eraCommonsID": {"type": "keyword"},
+            "bio": {"type": "text", "analyzer": "autocomplete"},
+            "story": {"type": "text", "analyzer": "autocomplete"},
+            "interests": {
+                "type": "text",
+                "analyzer": "autocomplete",
+                "fields": {"raw": {"type": "keyword"}},
+            },
+            "virtualStudies": {
+                "properties": {
+                    "id": {"type": "keyword", "index": False},
+                    "name": {
+                        "type": "text",
+                        "analyzer": "autocomplete",
+                        "fields": {"raw": {"type": "keyword"}},
+                    },
+                },
+                "type": "nested",
+            },
+            "searchableInterests": {
+                "type": "nested",
+                "properties": {
+                    "name": {
+                        "type": "text",
+                        "analyzer": "autocomplete",
+                        "fields": {"raw": {"type": "keyword"}},
+                    }
+                },
+            },
+        }
+    },
+}
+
+_MEMBERS_MAPPING = {
```

Comment:
This mapping is the same as the one above (legacy one) but `virtualStudies` field is omitted.